content/learning-paths/servers-and-cloud-computing/new-java/benchmarking.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Benchmarking via JMH 
+weight: 5
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Test the Java application for Performance 
+Now that you’ve built and run the Tomcat-like response, you can use it to test the JVM performance using JMH. You can also use it to test the performance difference between Cobalt 100 instances and other similar D series x86_64 based instances. 
```

Comment:
Add this line:
----------------------------------------
As noted before, the steps to benchmark remain the same, whether it's a Docker container or a custom VM.